cms/custom/interface-query-builder/src/query-builder/Filter.vue

```diff
@@ -0,0 +1,164 @@
+<template>
```

Comment:
Would it make sense to publish this as a npm package somehow?

Reply:
We can at least publish the query builder itself as an NPM package when we're happy with it. The directus-specific configuration has to be constructed like this to keep typing, vue capabilities and for directus to correctly register the extension.

For now, it's easier to have it in the same project.